compiler/il/OMRILOps.hpp

```diff
@@ -261,7 +261,9 @@ class ILOpCode
    bool isTwoChildrenAddress()  const { return isArrayRef(); }
    bool isCase()                const { return getOpCodeValue() == TR::Case; }
    bool isAnchor()              const { return getOpCodeValue() == TR::compressedRefs; }
-   bool isOverflowCHK()         const { return getOpCodeValue() == TR::OverflowCHK; }
+   bool isOverflowCHK()         const { return (getOpCodeValue() == TR::OverflowCHK)         || 
```

Comment:
it's currently used in an assert in OMR::Node::setNodeRequiresConditionCodes(bool v) to check if the node needs conditional check is reasonable. The assert checks "op.isArithmetic() || op.isLoadConst() || op.isOverflowCHK();" [https://github.com/eclipse/omr/blob/master/compiler/il/OMRNode.cpp#L8482](url)  The || here serves for this purpose.